components/31-molecules/opening-hours/_opening-hours.scss

```diff
@@ -100,7 +100,7 @@ $week-breakpoint: '(min-width: 750px)';
           align-items: center;
           justify-content: center;
           width: $month-day-size-mobile;
-          height: $month-day-size-mobile / 2;
+          height: $month-day-size-mobile *.5;
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.